content/docs/get-started/index.md

```diff
@@ -420,103 +362,94 @@ INFO:     Application startup complete.
+You'll also need to run [`heroku container:login`] to allow MLEM to push Docker
+images to Heroku's container registry.
 
-</details>
+[`heroku login`]:
+  https://devcenter.heroku.com/articles/heroku-cli#get-started-with-the-heroku-cli
+[`heroku container:login`]:
+  https://devcenter.heroku.com/articles/heroku-cli-commands#heroku-container
+[heroku api key]: http://heroku.com#
 
-To allow MLEM to push the Docker image built to the Heroku Docker Registry,
-you'll also need to execute `heroku container:login`.
+</admon>
 
-After getting authorized with Heroku, we can run the deployment command:
+After authenticating with Heroku, use `mlem deployment` (change `{myapp}`):
 
-```
+```cli
 $ mlem deployment run heroku app.mlem \
   --model models/rf \
-  --app_name {your-name}-mlem-get-started-app
+  --app_name {myapp}-mlem-get-started-app
 ⏳️ Loading model from models/rf.mlem
-⏳️ Loading deployment from app.mlem
-🛠 Creating docker image for heroku
-  🛠 Building MLEM wheel file...
-  💼 Adding model files...
-  🛠 Generating dockerfile...
-  💼 Adding sources...
-  💼 Generating requirements file...
-  🛠 Building docker image registry.heroku.com/{your-name}-mlem-get-started-app/web...
-  ✅  Built docker image registry.heroku.com/{your-name}-mlem-get-started-app/web
-  🔼 Pushing image registry.heroku.com/{your-name}-mlem-get-started-app/web to registry.heroku.com
-  ✅  Pushed image registry.heroku.com/{your-name}-mlem-get-started-app/web to registry.heroku.com
-🛠 Releasing app {your-name}-mlem-get-started-app formation
-✅  Service {your-name}-mlem-get-started-app is up. You can check it out at https://{your-name}-mlem-get-started-app.herokuapp.com/
+...
+  🛠 Building Docker image registry.heroku.com/{myapp}-mlem-get-started-app/web...
+  ✅  Built Docker image registry.heroku.com/{myapp}-mlem-get-started-app/web
+  🔼 Pushing image registry.heroku.com/{myapp}-mlem-get-started-app/web to registry.heroku.com
+  ✅  Pushed image registry.heroku.com/{myapp}-mlem-get-started-app/web to registry.heroku.com
+🛠 Releasing app {myapp}-mlem-get-started-app formation
+✅  Service {myapp}-mlem-get-started-app is up. You can check it out at https://{myapp}-mlem-get-started-app.herokuapp.com/
 ```
 
 A Deployment specification (or [declaration](/doc/command-reference/declare))
-was saved to `app.mlem`. Using this app declaration, you can re-deploy the same
-Heroku application with a different model.
+was written to the `app.mlem` metafile. You can use it to re-deploy the same
+Heroku app with a different model.
 
 <details>
 
-#### See app.mlem contents
+#### Click to see the contents of `app.mlem`.
 
 ```yaml
-$ cat app.mlem
 app_name: example-mlem-get-started-app
 object_type: deployment
 type: heroku
 ```
 
-</details>
+<admon type="info">
+
+Another special file was created: `app.mlem.state`. It contains metadata about
+the deployment, including which MLEM model we used, the URL of the deployment,
+and other useful information ([more info]).
```

Comment:
I'd delete that as well, but for now that file matters a lot and it's absence may cause problems. I assume we can remove it since the problem will be fixed https://github.com/iterative/mlem/issues/463